src/v/config/configuration.cc

```diff
@@ -701,7 +701,44 @@ configuration::configuration()
         model::fetch_read_strategy::polling,
         model::fetch_read_strategy::non_polling,
         model::fetch_read_strategy::non_polling_with_debounce,
+        model::fetch_read_strategy::non_polling_with_pid,
       })
+  , fetch_pid_p_coeff(
+      *this,
+      "fetch_pid_p_coeff",
+      "Proportional coefficient for fetch PID controller.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      100.0,
+      {.min = 0.0, .max = std::numeric_limits<double>::max()})
+  , fetch_pid_i_coeff(
+      *this,
+      "fetch_pid_i_coeff",
+      "Integral coefficient for fetch PID controller.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      0.01,
+      {.min = 0.0, .max = std::numeric_limits<double>::max()})
+  , fetch_pid_d_coeff(
+      *this,
+      "fetch_pid_d_coeff",
+      "Derivative coefficient for fetch PID controller.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      0.0,
+      {.min = 0.0, .max = std::numeric_limits<double>::max()})
+  , fetch_pid_target_utilization(
```

Comment:
Should add a suffix too. Not sure which since there's still discussion if percent/fraction

Reply:
It'll be a fraction, is there a specific suffix we're uses for those? `_fraction`?

Reply:
Added the `_fraction` suffix to it as discussed on Slack.